src/2_RTB/PianoDiQualifica/content.typ

```diff
@@ -164,14 +165,473 @@ Sono state individuate due metriche principali:
+      ],"N/I","ROF"),
+      ([
+            Verificare che un sensore possa inserire nel sistema dati relativi congestione della strada, controllata dal sensore, con annesso coordinate e #glossary("timestamp") della misurazione.
+      ],"N/I","ROF"),
+      /*TODO: manca test (poichè manca requisito relativo) che vada a controllare che il senosre annetta anche dati relativi al suo stato personale */
+).enumerate(start:1).map(test => ("TS"+ str(test.at(0)),test.at(1).at(0),test.at(1).at(1),test.at(1).at(2))).enumerate(start:1).map(test => (test.at(1).at(0),test.at(1).at(1),test.at(1).at(2),test.at(1).at(3) + str(test.at(0))));
+
+#show figure: set block(breakable: true)
+#figure(
+table(
+      columns: (2fr,7fr,1fr),
+      align: (center,left,center),
+      fill:(_,row) =>if row==0 {luma(150)} else if calc.odd(row) { luma(220)} else {white},
+      [*Codice test*],[*Descrizione*],[*Stato*],
+      ..test_di_sistema.map(item => (item.at(0),item.at(1),item.at(2))).flatten().map(item => [#item])
+),caption: "Test di sistema.")
+
+=== Tracciamento dei test di sistema
+#show figure: set block(breakable: true)
+#figure(
+table(
+      columns: (80pt,80pt),
+      align: (center,center),
+      fill:(_,row) =>if row==0 {luma(150)} else if calc.odd(row) { luma(220)} else {white},
+      [*Codice test*],[*Codice caso d'uso*],
+      ..test_di_sistema.map(item => (item.at(0),item.at(3))).flatten().map(item => [#item])
+),caption: "Tracciamento dei test di sistema.")
 
 
+== Liste di controllo
 
+Le liste di controllo sono uno strumento che i verificatori possono utilizzare al fine di individuare errori ricorrenti nella documentazione. Tali liste prevedono anche una descrizione del problema al fine di poter fornire una spiegazione della richiesta di cambiamenti durante la fase di revisione.
+Le liste di controllo possono essere aggiornate durante tutto il corso del progetto dai Verificatori, man mano che vengono notati errori ricorrenti.
 
+#pagebreak()
+=== Struttura dei documenti
+
+#figure(
+table(
+      columns: (1fr, 2fr),
+      inset: 10pt,
+      align: (center, left),
+      [*Aspetto*],[*Spiegazione*],
+      [Vuoti documentativi],[Tutte le sezioni che non si intende completare, e che quindi devono rimanere vuote, devono essere cancellate.],
+      [Didascalia assente],[Tutte le tabelle e le immagini devono avere una didascalia descrittiva.],
+      [Ripetizione di elementi ricorrenti],[Ogni elemento che viene usato più di una volta, soprattutto se si sa a priori che verrà usato più volte, deve essere creato da una funzione #glossary("Typst").],
+      [Ordine non alfabetico],[I nomi dei componenti devono essere riportati in ordine alfabetico, in qualsiasi documento dove compaiono come elenco non numerato.],
+      [Aggiornamento fantasma],[Ad ogni insieme di modifiche ai documenti che devono essere aggiunte assieme, deve corrispondere una riga nella tabella del changelog, con un univoco numero di versione.],
```

Comment:
sincero non si capisce molto bene cosa bisogna cotrollare

Reply:
Non capisco a cosa ti stai riferendo.

Reply:
Semplicemente vuol dire che ad ogni "batch" di modifiche apportate ad un documento, deve corrispondere una nuova minor version nel changelog come riga.


In sostanza **NON ESISTE** che uno aggiorna un documento senza aggiornare il changelog.

Reply:
quindi si potrebbe  mettere il punto sul fatto che il verificatore deve controllare che ci sia almeno una nuova linea relativa alle modifiche nel changelog

Reply:
esattamente quanto detto in tale riga ma in modo più formale e dettagliato.